app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -9,19 +9,21 @@ def textual_group_properties
       creation_timestamp
       resource_version
       session_affinity
+      service_type
       portal_ip
     )
   end
 
   def textual_group_port_configs
-    labels = [_("Name"), _("Port"), _("Target Port"), _("Protocol")]
+    labels = [_("Name"), _("Port"), _("Target Port"), _("Protocol"), _("Node Port")]
     h = {:labels => labels}
     h[:values] = @record.container_service_port_configs.collect do |config|
       [
         config.name || _("<Unnamed>"),
+        config.protocol,
```

Comment:
This was supposed to be part of [1] but I forgot to reorder the labels. Fixed.

[1] "reorder columns to match k8b v1 types.go order."
Name, Protocol, Port, Target Port